src/app/component/ExpandableContentShowMore/ExpandableContentShowMore.component.js

```diff
@@ -70,11 +99,24 @@ export class ExpandableContentShowMore extends PureComponent {
         const { children, showElemCount } = this.props;
         const { isOpen } = this.state;
 
-        const child = !isOpen ? children.slice(0, showElemCount) : children;
+        const childStatic = children.slice(0, showElemCount);
+        const childExpandable = children.slice(showElemCount);
+
+        const style = {
+            maxHeight: isOpen ? this.expandableContentHeight : 0
+        };
 
         return (
             <>
-                { child }
+                { childStatic }
+                <div
+                  ref={ this.expandableRef }
+                  block="ExpandableContentShowMore"
+                  elem="ExpandableChildren"
+                  style={ style }
+                >
+                    { childExpandable }
+                </div>
```

Comment:
I think this can actually slow down the page in case there are more then 20 options in 20 attributes, etc. Is there a way around this render? Animating new elements which appear on page could be hard, but hmm.

Reply:
Commited a fix for it :)